### PR TITLE
VEN-428 | Add Status to BerthApplication

### PR DESCRIPTION
**Pull request by a-rmz**
## Description ✨ 
* Add a `Status` enum with different handling status for applications.
* Add a `status` field to the `BerthApplication` model

## Issues 🐛 
Closes [VEN-428](https://helsinkisolutionoffice.atlassian.net/browse/VEN-428)

## Testing ⚗️ 
```graphql
query GetApplications {
  berthApplications {
    edges {
      node {
        id
        status
      }
    }
  }
}
```
Should return `"PENDING"`

### CODE REVIEW

File: applications/enums.py [hunk 1]
```diff
@@ -11,3 +11,10 @@ class Labels:
         ON_TRESTLES = _("On trestles")
         ON_TRAILER = _("On a trailer")
         UNDER_TARP = _("Under a tarp")
+
+
+class ApplicationStatus(Enum):
+    PENDING = "pending"
+    OFFER_GENERATED = "offer_generated"
+    OFFER_SENT = "offer_sent"
+    HANDLED = "handled"
```
Comment (igordavydsson) [inline applications/enums.py]: please, add `Labels`

Comment (a-rmz) [reply applications/enums.py]: Isn't that gonna be handled by the front-end?

File: applications/models.py [hunk 2]
```diff
@@ -150,6 +150,14 @@ class BerthApplication(BaseApplication):
         related_name="berth_applications",
     )
 
+    status = EnumField(
+        ApplicationStatus,
+        null=False,
```
Comment (igordavydsson) [inline applications/models.py]: `null=False` is already default

File: applications/enums.py (hunk 1 above)
Comment (igordavydsson) [reply applications/enums.py]: these `Labels` are for the django-admin 🙂

Comment (a-rmz) [reply applications/enums.py]: Aha! Got it. :)

**Pull request merged**